### ISSUE TITLE
encoder.go error

### ISSUE THREAD

**Issue by wyunshaw** [googollee/go-socket.io#201]
go-socket.io/parser/encoder.go
when i = 10,  will write to be ":";

```go
func (e *Encoder) writeUint64(w byteWriter, i uint64) error {
	base := uint64(1)
	for i/base > 10 {
		base *= 10
	}
	for base > 0 {
		p := i / base
		if err := w.WriteByte(byte(p) + '0'); err != nil {
			return err
		}
		i -= p * base
		base /= 10
	}
	return nil
}
```

when  changed like followed code,will run well; 
```go  
for i/base >= 10 {
    base *= 10
}
```

**Comment by erkie** [comment]
@wyunshaw could you please make a PR for this?

**Comment by mrfoe7** [comment]
Resolved by #251 PR
